compiler/testData/diagnostics/tests/extensions/throwOutCandidatesByReceiver.fir.kt

```diff
@@ -8,7 +8,7 @@ fun <R> List<R>.a() {}
 
 fun test1(i: Int?) {
     1.<!INAPPLICABLE_CANDIDATE!>a<!>()
```

Comment:
Just for the record, many errors in this file should be rather `UNRESOLVED_REFERENCE_WRONG_RECEIVER`: https://github.com/JetBrains/kotlin/blob/master/compiler/testData/diagnostics/tests/extensions/throwOutCandidatesByReceiver.kt#L10